src/main/java/seedu/intrack/model/ModelManager.java

```diff
@@ -22,6 +22,7 @@ public class ModelManager implements Model {
     private final InTrack inTrack;
     private final UserPrefs userPrefs;
     private final FilteredList<Internship> filteredInternships;
+    private final FilteredList<Internship> selectedInternship;
```

Comment:
```suggestion
    private final FilteredList<Internship> selectedInternships;
```
Perhaps this should be plural since it is a ```List```?